rules/ujson.bzl

```diff
@@ -29,7 +29,8 @@ def _ujson_rust(ctx):
     # 3. Substitute all `rust_attr` for `#`, thus creating rust attributes.
     # 4. Format it with `rustfmt` so it looks nice and can be inspected.
     command = """
-        {preprocessor} -nostdinc -I. -DRUST_PREPROCESSOR_EMIT=1 -DNOSTDINC=1 {defines} $@ \
+        {preprocessor} -nostdinc -I. -Iexternal/lowrisc_opentitan \
```

Comment:
There may be a better way to do this: examine the ujson_lib paths and derive the root path from there.  With that path in had, we could eliminate both `-I. -Iexternal/lowrisc_opentitan` in favor of `-I{ujson_root_path}`.

Reply:
Great idea! Let me try it out and send a new patch. Thanks!